src/components/Modals/SaveAsModal.vue

```diff
@@ -35,7 +35,13 @@ export default {
   methods: {
     downloadFile: function() {
       const a = document.createElement('a');
-      a.setAttribute('href', `data:text/json;charset=utf-8,${encodeURIComponent(JSON.stringify(this.dataToDownload))}`);
+      const blob = new Blob(
+        [JSON.stringify(this.dataToDownload)],
+        {
+          type: 'text/plain;charset=utf-8',
```

Comment:
I think this still wants to be `text/json`. Probably it doesn't matter too much, but we do know that it's json so we may as well set the mimetype to the right value.